### DISCUSSION TITLE
Question about Shadowsocks 2022 safety.

### DISCUSSION THREAD

**Discussion by nDman** [XTLS/Xray-core#1570] [General]
Is it enough secure to be used instead of (vless, vmess, trojan)? 
In Iran government started to block proxy-vpn domains. Two of my domains blocked now,  I can connect to my servers with their IP but I can't open domain. I made RDNS for my Bind server, I don't know how they can find ip's domains. maybe they probing dns transactions. 
So I thinks having a server behind a CDN and domain is pointless, I like to use a proxy independent from domains and certificate files.

Is Shadowsocks 2022 safe to be used?
What is best cipher to use for it?

**Comment by Smallthing** [comment]
no 
it's not safe

**Reply by nDman** [reply]
Yeah I found that by testing it, thanks anyway.
